lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerController.pm

```diff
@@ -21,9 +21,11 @@ use parent 'Installation::Partitioner::LibstorageNG::v4::ExpertPartitionerContro
 use Installation::Partitioner::LibstorageNG::v4_3::AddLogicalVolumePage;
 use Installation::Partitioner::LibstorageNG::v4_3::AddVolumeGroupPage;
 use Installation::Partitioner::LibstorageNG::v4_3::ClonePartitionsDialog;
+use Installation::Partitioner::LibstorageNG::v4_3::ConfirmationWarning;
```

Comment:
I guess we can delete SmallForSnapshotsWarning implemented [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11501/files#diff-0ffd80c95fb51756da5f9d8a80e01b4f6c2a14898cf18aa594456bc79cccffcdR26) and check if all is ok in [splitusr](https://openqa.opensuse.org/tests/1502515)

Reply:
I also was thinking that it may be a good solution to remove all the classes for the certain warnings (all of them are just empty) and use ConfirmationWarning instead. As of now that solves all the purposes (press yes/no, check label). In case we'll need to have some specific methods for some certain warning in feature, we could create the separate class for it then. WDYT?

Those Warning classes are:
- DeletingCurrentDevicesWarning;
- ModifiedDevicesWarning;
- SmallForSnapshotsWarning.

Reply:
I guess the original idea was not to have those label as test data and have them as part of the Page, as the page is something static, ideally it should be part of the creation of that ConfirmationWarning Page, but I'm might be wrong. In any case I agree that all are the same, we could have just one class for them.

Reply:
Having labels as part of the page also makes sense. The only cons in the case, we'll have to create separate `confirm_warning` and `is_shown` methods in the controller for each warning. But as there are a lot of warnings exist, there will be a lot of such methods in ExpertPartitionerController. Maybe, in that case we'll need to create some separate controller for warnings and access it through ExpertPartitionerController. Something like `get_expert_partitioner_controller->get_warnings_controller()->is_shown_small_for_snapshot_warning()`.

Reply:
yes, the idea would be to change it only in the code once, not for each test suite as test data.
I think we would just need it as part of the `sub confirm_warning` without any `is_show` so it would be just part of the flow of the program, not something specific to check in the test, like it is the case in that other test suite I mentioned (for example, we have some actions and in the middle some warning appears and later other actions).
That new controller probably makes sense if they are many warning in your current task.
Please file a ticket to not forget (for now we have duplicated that code).